bedrock/base/templates/includes/protocol/navigation/menu-firefox/about.html

```diff
@@ -0,0 +1,72 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% set referral="?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=nav&amp;utm_content=firefox" %}
+
+<li class="mzp-c-menu-category mzp-has-drop-down mzp-js-expandable">
+  <a class="mzp-c-menu-title" href="{{ url('mozorg.about') }}" aria-haspopup="true" aria-controls="mzp-c-menu-panel-about">{{ _('About') }}</a>
+  <div class="mzp-c-menu-panel mzp-has-card" id="mzp-c-menu-panel-about">
+    <div class="mzp-c-menu-panel-container">
+      <button class="mzp-c-menu-button-close" type="button" aria-controls="mzp-c-menu-panel-about">{{ _('Close About menu') }}</button>
+      <div class="mzp-c-menu-panel-content">
+        <ul>
+          <li>
+            <section class="mzp-c-menu-item mzp-has-icon">
+              <a class="mzp-c-menu-item-link" href="https://blog.mozilla.org/firefox/{{ referral|safe }}" data-link-name="Firefox Blog" data-link-type="nav" data-link-position="topnav" data-link-group="about">
```

Comment:
Not necessarily part of this PR, but the Firefox blog is in English, German, and French, but it **doesn't** do automatic detection. I think we should set this link up with a helper array like we do for the press blogs so we can output a direct link to each language (noting this mostly for myself).